crates/jstz_cli/src/deploy.rs

```diff
@@ -1,25 +1,59 @@
+use dialoguer::Confirm;
 use jstz_proto::{
     operation::{Content, DeployFunction, Operation, SignedOperation},
     receipt::Content as ReceiptContent,
 };
 use log::{debug, info};
 
 use crate::{
+    account,
     config::{Config, NetworkName, SmartFunction},
     error::{bail, bail_user_error, user_error, Result},
+    sandbox::daemon,
+    term::styles,
     utils::read_file_or_input_or_piped,
 };
 
 pub async fn exec(
-    code: Option<String>,
+    code_op: Option<String>,
     balance: u64,
     name: Option<String>,
     network: Option<NetworkName>,
 ) -> Result<()> {
     let mut cfg = Config::load()?;
 
-    let (_, user) = cfg.accounts.current_user().ok_or(user_error!(
-        "You are not logged in. Please run `jstz login`."
+    // Load sandbox if network is Dev or it is None and cfg network is dev, and sandbox is not already loaded
+    if (network == Some(NetworkName::Dev)
+        || (network.is_none() && cfg.networks.default_network == Some(NetworkName::Dev)))
+        && cfg.sandbox.is_none()
```

Comment:
I believe there should be a function in `Config` for getting the current network name

Reply:
there is currrently no function like this - I've added a new one